calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+| `max_initial_calibration_translation_error` | `double`      | `1.0`                                                   | Maximum allowable translation error in meters in the calibration process. If this error exceeds the specified value, a warning message will appear in the console. |
+| `max_initial_calibration_rotation_error`    | `double`      | `45.0`                                                  | Maximum allowable rotation error in degrees in the calibration process. If this error exceeds the specified value, a warning message will appear in the console.   |
+| `max_number_of_combination_samples`         | `int`         | `10000`                                                 | Maximum number of samples from combinations that are used for cross-validation during the calibration process.                                                     |
+
+## Requirements
+
+### Radar reflector
+
+The type of reflector shown in the image below is crucial for our calibration because it has a highly predictable and consistent response to radar waves. The triangular shape, often composed of three metal plates arranged in a prism form, ensures that the reflector returns signals in specific, predictable ways.
+
+It is recommended that the user build the radar reflector on a tripod, securing it with tape to ensure stability. Additionally, nothing should be attached above the radar reflector; it must be the highest object on the entire calibration target. Additionally, make sure the height of the radar reflector is not larger than the `reflector_max_height` parameter.
+
+<p align="center">
+    <img src="../docs/images/marker_radar_lidar_calibrator/radar_reflector.png" alt="radar_reflector" width="150">
+<p align="center">
+
+## Known issues/limitations
+
+- While extracting the background model, ensure that the radar reflectors are not in the calibration area and that no one is moving around the calibration area.
+
+- While performing the calibration, the calibrator provides a button to delete any mismatched pairs (e.g., an object detected by both radar and lidar). However, some outliers may not be easily detectable by human eyes, leading to imperfect results as the calibration proceeds even with these anomalies present. Future enhancements will aim to improve outlier detection, thereby refining the calibration accuracy.
+
+- The calibrator should be able to handle different lidar and radar sensors. So far, We calibrated Velodyne VLS-128 lidar sensor, Pandar-40P lidar sensor, and ARS408 radar sensor with good calibration results.
+
+## Pro tips/recommendations
+
+- While performing the calibration, it is required that all radar reflectors and the radar itself lie on the same plane. The lidar does not have the same limitation.
+- During calibration, place the reflectors at various distances and ensure that the center of the radar reflector faces the radar sensor.
+- The calibration area is recommended to setup like the image below.
```

Comment:
Please re write this one

Reply:
rewrite in https://github.com/tier4/CalibrationTools/pull/164/commits/087fa2970d143a29cfacb28cf0b7d25b268a412d

Reply:
You mention different angles and at the same that the must be facing the radar.
I know what you mean, but the way you wrote is contradictory.

Rewritten*

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/b432423e6103a705f501b240359ddf41c905ddce

Reply:
The new phrasing is also not correct.
If you say at different distances, the use could very well place several corner reflectors in a straight line
The previous text mentioning the angle was wrong since you did not explain which angle were you referring to
A better way to do it would be mentioning different positions within the radar+lidar common FOV, trying to cover as much of it

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/2d5b949782c0793423b89ffb3fa0ca28e61b5296